controllers/FrontController.php

```diff
@@ -170,9 +170,10 @@ protected function getCurrentCart()
      * @param $cart
      * @param $customer
      * @param $isAjax
+     * @param $cancelled
      * @throws AdyenException
      */
-    protected function handlePaymentsResponse($response, $cart, $customer, $isAjax)
+    protected function handlePaymentsResponse($response, $cart, $customer, $isAjax, $cancelled = false)
```

Comment:
Please double check if the paypal payments/details response is going to be indeed Cancelled or Refused, in case it's cancelled please just use the resultCode and you don't need the cancelled flag from the frontend